docs-conceptual/azureadps-2.0/Cannot-find-object.md

```diff
@@ -35,12 +38,13 @@ Get-AzureADUser -All $True
 Now all user objects are returned, and the filter works on all user objects in your directory. So the correct way to find all users where City equals London is:
 
 ```powershell
-Get-AzureADUser -All $True | Where ($_.City -eq "London")
+Get-AzureADUser -All $True | Where {$_.City -eq "London"}
 ```
 
-Note that this approach works for all properties of objects. For a limited number of string properties you can also use the -SearchString parameter, as in
+Note that this approach works for all properties of objects. For a limited number of string properties you can also use the `-SearchString` parameter. For example,
 
 ```powershell
 Get-AzureADUser -SearchString "London"
 ```
-This will return all users where one of the qualifying string attributes on the user object equals "London" - which would include users who's DisplayName equals "Jack London" or where the Department equals "London".
+
+This will return all users where one of the qualifying string attributes on the user object equals `London` - which would include users who's DisplayName equals `Jack London` or where the Department equals `London`.
```

Comment:
```suggestion
This will return all users where one of the qualifying string attributes on the user object equals `London`, which would include users whose DisplayName equals `Jack London` or where the Department equals `London`.
```